analytics/client.go

```diff
@@ -205,3 +295,42 @@ func (d disabledLogger) Logf(format string, args ...any) {
 func (d disabledLogger) Errorf(format string, args ...interface{}) {
 	// Do nothing.
 }
+
+// A custom segment logger that logs to glog.
+type amplitudeLogger struct{}
```

Comment:
This file is starting to feel a bit cluttered. We should separate the code for Mixpanel, Segment, and Amplitude out into separate files. **Non-blocking.**

Reply:
I have not done it cause we might be removing the segment and Mixpanel SDKs when we are sure that Amplitude is working fine and all the metrics are passed to Amplitude.